python/lsst/meas/extensions/trailedSources/NaivePlugin.py

```diff
@@ -191,3 +201,61 @@ def fail(self, measRecord, error=None):
             self.flagHandler.handleFailure(measRecord)
         else:
             self.flagHandler.handleFailure(measRecord, error.cpp)
+
+    def computeSecondMomentDiff(self, z, c):
```

Comment:
If this method only exists to be used internally in the other method, consider marking it 'private' with a leading underscore i.e. `_computeSecondMomentDiff` to mark that it is implementation detail, and not a stable interface. Doing that frees you up to change the behavior later if something comes up, without fear that someone somewhere is depending on the existing behavior.